### PR TITLE
Fix: #4524 - Add Release notes for v1.32.3

### LINKED ISSUE #4524: Release Notes for iOS Release v1.32.3
- Replaced our HTTPSE implementation with Apple's HTTPS upgrades in WKWebView. ([#4335](https://github.com/brave/brave-ios/issues/4335))
- Updated designs on error and interstitial pages. ([#4338](https://github.com/brave/brave-ios/issues/4338))
- Fix app crash when closing Brave via app switcher. ([#4490](https://github.com/brave/brave-ios/issues/4490))

### PR DESCRIPTION
**Pull request by srirambv**
## Summary of Changes

This pull request fixes #4524

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Test Plan:

NA

## Screenshots:

NA

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by kjozwiak — APPROVED** [review]
LGTM 👍 Nice work!

**Comment by srirambv** [comment]
Will merge once the issue are all verified and signed off

**Pull request merged**
